package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fastify-accepts-serializer",
-  "version": "3.0.1",
+  "version": "3.1.0",
```

Comment:
Do not modify the package version in PRs.

Reply:
actually, it is merge from upstream. not modify by me.

Reply:
`git checkout master && git pull && git checkout your-branch && git rebase master`

Reply:
A little bit late, I already removed the repo.... I will reopen another one. 
Thanks for your suggestion, I will use it another time.